public/i18n/en/translation.json

```diff
@@ -307,7 +317,42 @@
       }
     },
     "backup_and_restore": {
-      "title": "Backup & restore"
+      "title": "Backup & restore",
+      "tabs": {
+        "backup": "Backup",
+        "restore": "Restore"
+      },
+      "backup": {
+        "description": "Download your firewall's current settings and data. This will serving as a safeguard against potential data loss or system failures. Configure a passphrase to increase the security of your data.",
+        "description_entrprise": "Run a backup to download your firewall's current settings and data. The system runs autonomously and keeps up to 3 backups. Configure a passphrase to increase the security of your data.",
+        "download": "Download",
+        "download_backup": "Download backup",
+        "modal_download_title": "Downloaded backup",
+        "modal_download_description": "Are you sure you want to proceed with the '{name}' backup?",
+        "passphrase_drawer_title": "Configure passphrase",
+        "passphrase": "Passphrase",
+        "passphrase_helper": "Lorem ipsum..",
+        "configure_passphrase": "Configure passphrase",
+        "no_backups_found": "No backups found",
+        "run_backup": "Run backup",
+        "run_backup_description": "Are you sure you want to proceed with the '{name}' backup?",
+        "success_download": "Backup downloaded successfully"
+      },
+      "restore": {
+        "description": "Restore the firewall's previous configuration and settings from a saved backup file, ensuring it functions with its original rules and parameters.",
+        "restore": "Restore",
+        "restore_now": "Restoring now",
+        "restore_in_progress": "Restoring backup in progress",
+        "restore_backup": "Restore backup",
+        "passphrase": "Passphrase",
+        "passphrase_helper": "Lorem ipsum..",
```

Comment:
```suggestion
        "passphrase_helper": "Enter the passphrase used to encrypt the backup; if the passphrase is incorrect, it will not be possible to restore the backup.",
```